pliers/utils/updater.py

```diff
@@ -59,12 +57,11 @@ def check_updates(transformers, datastore=None, stimuli=None):
                 res = trans.transform(stim)
 
                 try: # Add iterable
-                    res = [res._data for r in res]
+                    res = [getattr(res, '_data', res.data) for r in res]
                 except TypeError:
-                    res = res._data
+                    res = getattr(res, '_data', res.data)
 
-                res = hash_data(res) if isinstance(
-                    trans, (Converter, Filter)) else res[0][0]
+                res = hash_data(res)
```

Comment:
Does this work on numpy arrays (e.g. from a LibrosaFeatureExtractor)? JSON (e.g. from a ClarifaiAPIExtractor)?